js/ui/popup/popup_position_controller.js

```diff
@@ -46,7 +43,7 @@ class PopupPositionController extends OverlayPositionController {
     }
 
     get outsideDragFactor() {
-        return this._outsideDragFactor;
+        return this._getOutsideDragFactor();
```

Comment:
seems like we can get rid of internal getter and just write the content of "_getOutsideDragFactor" here